### PR TITLE
updated README.md

### PR DESCRIPTION
**Pull request by SanyuktaSaha**
Basically I added extra lines of code which can possibly be used in different operating systems.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,15 +109,20 @@ $ git clone https://github.com/paritytech/parity-ethereum
 $ cd parity-ethereum
 
 # build in release mode
-$ cargo build --release --features final
+$ cargo build --release -p features final
```
Comment (niklasad1) [inline README.md]: this is wrong,  `-p` is used to specify which crate to build, `--features` is something different.

Please revert the change

File: README.md [hunk 2]
```diff
@@ -87,15 +109,20 @@ $ git clone https://github.com/paritytech/parity-ethereum
...
+```$ git submodule init```
+```$ git submodule update```
 
 This produces an executable in the `./target/release` subdirectory.
 
 Note: if cargo fails to parse manifest try:
 
 ```bash
 $ ~/.cargo/bin/cargo build --release
+$ ./target/release/features final --help
```
Comment (niklasad1) [inline README.md]: this is also wrong the `binary` will be named `parity`.

there's no CLI option `final` is just a feature flag AFAIK

File: README.md [hunk 3]
```diff
@@ -79,6 +87,20 @@ Once you have `rustup` installed, then you need to install:
...
 
+or
+
+ You can build Parity Ethereum from the sources; you can launch Parity Ethereum using the binaries built for the latest releases for Linux, Mac/Homebrew and Windows or, if you’re on an Ubuntu Snappy platform, just use our Snappy App.
+ 
+ For Linux systems:
+
+Ubuntu, Debian
+
+  ```$ apt-get install build-essential cmake libudev-dev```
```
Comment (niklasad1) [inline README.md]: I don't think `libudev-dev` is required anymore since we removed `hardware wallet` support

File: .dockerignore [hunk 4]
```diff
@@ -6,6 +6,7 @@ target
 *.swo
 *.swn
 *.DS_Store
+*.md
```
Comment (debris) [inline .dockerignore]: this is wrong
